frontend/src/modules/profile/view.jsx

```diff
@@ -3,14 +3,15 @@ import { Button, notification, Avatar, Menu, Row, Col } from "antd";
 import React, { useRef, useState, useEffect } from "react";
 import StickyBox from "react-sticky-box";
 import api from "../../utils/api";
+
+//import SignupForm from "../stakeholder-signup/signup-form";
```

Comment:
The /profile page would be currently broken, with this commented out. I feel a little uncomfortable deploying this to test, since it would mean other things would not be testable.